scripts/monitoring/cron-send-project-stats.py

```diff
@@ -253,9 +253,11 @@ def main():
                 current_time = datetime.datetime.now()
                 time_keeps = current_time - old_time
                 print 'the project in Terminating status for %s' % time_keeps
-
-                time_keeps_max = max(time_keeps_max, (current_time - old_time).seconds)
-
+                if current_time > old_time:
+                    time_keeps_max = max(time_keeps_max, (current_time - old_time).seconds)
```

Comment:
Why (current_time - old_time).seconds ? When line 254 is the same claulation?

Why not change to time_keeps.seconds?